examples/output_directory/xtb_out/energy.output

```diff
@@ -67,20 +67,20 @@
    J. Pisarek, P. Pracht, J. Seibert, P. Shushkov, S. Spicher, M. Stahn
    T. Strunk, J. Stückrath, T. Rose, and J. Unsleber
    
- * started run on 2021/05/05 at 13:43:00.598     
+ * started run on 2021/07/21 at 15:57:58.503     
 
            -------------------------------------------------
           |                Calculation Setup                |
            -------------------------------------------------
 
-          program call               : /home/atarzia/software/xtb-6.4.0/bin/xtb /home/atarzia/software/stk_optimizers/examples/output_directory/xtb_out/input_structure.xyz --gfn 2 --parallel 1 --etemp 300 --chrg 0 --uhf 0
```

Comment:
As this contains hard-coded paths, can this instead be renamed as example_energy.output? The user who runs the code would not receive exactly the same output file